### PR TITLE
This is so Sad

### PR DESCRIPTION
**Pull request by Tswinston7**
@twang I was working on Much Better, and although my code work, it didn't work completely on the final product. What was the mistake for this error?

### CODE REVIEW

File: Test.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+public class MuchBetter{
+
+    public static void main(String[] args){
+    
+        System.out.println("A \"quoted\" String is \n'much' better if you learn \nthe rules of \"escape ");
+        System.out.print("sequences.\" \nAlso, \"\" represents an empty String. \nDon't forget: use");
```
Comment (marquss) [inline Test.txt]: \n before "Also" makes answer wrong.

File: Test.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+public class MuchBetter{
+
+    public static void main(String[] args){
+    
+        System.out.println("A \"quoted\" String is \n'much' better if you learn \nthe rules of \"escape ");
```
Comment (marquss) [inline Test.txt]: \n after "learn" makes answer wrong.

File: Test.txt (hunk 1 above)
Comment (marquss) [inline Test.txt]: "Sequences" should have been part of the last print statement.

**Comment by twang** [comment]
Thanks @Tswinston7 and @marquss!

**Pull request closed without merging**